config/locales/hyrax.en.yml

```diff
@@ -11,6 +11,45 @@ en:
           based_near: Location
           contributor: Contributors
           keyword: Keyword
+  datetime:
```

Comment:
It is added to allow for the spanish translations for time_ago_in_words (as you probably saw since you commented below in the view as well).